solutions/cses-1093.mdx

```diff
@@ -2,18 +2,76 @@
 id: cses-1093
 source: CSES
 title: Two Sets II
-author: Sofia Yang
+author: Sofia Yang, Ryan Chou
 ---
-[Official Editorial](https://codeforces.com/blog/entry/70018)
+[Official Editorial (C++)](https://codeforces.com/blog/entry/70018)
 
-## Solution
+## Explanation
 
-**Time Complexity:** $\mathcal{O}(N^3)$
+Let's rule a simple case first. When can you not construct a valid solution?
+
+Since the two sets have to have an equal sum, it's evident that both of them have to have half of  $1, 2, \ldots, N$. So, if $\frac{N(N+1)}{2}$, or the sum of the first $N$ positive integers isn't even, then a solution isn't possible. For each subset, we want it's sum to be half of this sum, or $\frac{N(N+1)}{4}$
```

Comment:
```suggestion
Since the two sets have to have an equal sum, it's evident that both of them have to have half of  $1, 2, \ldots, N$. So, if $\frac{N(N+1)}{2}$, or the sum of the first $N$ positive integers isn't even, then a solution isn't possible. For each subset, we want its sum to be half of this sum, or $\frac{N(N+1)}{4}$
```

Reply:
mb